cmd/notary/tuf.go

```diff
@@ -619,9 +625,16 @@ func (t *tufCommander) tufReset(cmd *cobra.Command, args []string) error {
 	}
 
 	if t.resetAll {
-		return cl.Clear(t.archiveChangelist)
+		err = cl.Clear(t.archiveChangelist)
+	} else {
+		err = cl.Remove(t.deleteIdx)
 	}
-	return cl.Remove(t.deleteIdx)
+	// If it was a success, print to terminal and return nil
+	if err == nil {
+		cmd.Printf("Successfully reset specified changes for repository %s\n", gun)
+		return nil
```

Comment:
I suppose this line could be removed? If so the comment at line 632 should be updated.